src/acme/ui/CustomerCreatePanel.java

```diff
@@ -0,0 +1,144 @@
+package acme.ui;
+
+import java.util.Collections;
+import java.util.Set;
+
+import javax.persistence.NoResultException;
+
+import acme.data.PersistableEntity;
+import acme.pd.Company;
+import acme.pd.Customer;
+import acme.pd.Road;
+import acme.pd.User;
+import javax.swing.JLabel;
+import java.awt.Font;
+import javax.swing.JTextField;
+import javax.swing.JComboBox;
+import javax.swing.JButton;
+
+public class CustomerCreatePanel extends AcmeBaseJPanel {
+	Company company;
+	Customer customer;
+	private JTextField nameField;
+	private JLabel numberField;
+	private JComboBox<Road> avenueComboField;
+	private JComboBox<Road> streetComboField;
+
+	public CustomerCreatePanel() {
+        super();
+        customer = new Customer();
+        customer.create();
+    }
+	
+	@Override
+    public void buildPanel() {
+        company = this.getCompany();
+        this.removeAll();
+        
+        setLayout(null);
+        
+        JLabel titleLabel = new JLabel("Customer");
+        titleLabel.setBounds(42, 30, 75, 22);
+        titleLabel.setFont(new Font("Tahoma", Font.PLAIN, 18));
+        add(titleLabel);
+        
+        JLabel nameLabel = new JLabel("Name");
+        nameLabel.setBounds(57, 88, 46, 14);
+        add(nameLabel);
+        
+        nameField = new JTextField();
+        nameField.setBounds(113, 85, 125, 20);
+        add(nameField);
+        nameField.setColumns(10);
+        
+        JLabel numberLabel = new JLabel("Number");
+        numberLabel.setBounds(57, 126, 46, 14);
+        add(numberLabel);
+        
+        
+        numberField = new JLabel(Integer.toString(Customer.getNextCustomerNumber()));
+        numberField.setBounds(113, 126, 46, 14);
+        add(numberField);
+        
+        JLabel lblIntersection = new JLabel("Intersection");
+        lblIntersection.setFont(new Font("Tahoma", Font.PLAIN, 14));

```

Comment:
instead of randomly setting fonts to tahoma, lucida, etc. It would be better to just call (lablIntersection.getFont().toString()) here so that the font remains the same, you're just changing the size.